CMakeLists.txt

```diff
@@ -610,6 +610,34 @@ if (ENABLE_TESTS)
 
 endif()
 
+add_custom_target(tags COMMAND
+  cd ${CMAKE_CURRENT_SOURCE_DIR} && ctags
+    --c++-kinds=+p
+    --exclude=build
+    --exclude=.ci
+    --exclude=cmake
+    --exclude=cmake_templates
+    --exclude=cmake_templates
+    --exclude=debian
+    --exclude=debian
```

Comment:
dupe

Reply:
*sigh* in the good old days, I would have just removed those two lines following up your commit and this would have been done already.  now the tests are not even complete.  and I have to wait for you to apply the change - before I can re-review  - if I'm not at something completely different by then.

Reply:
I feel your pain, I've removed the duplication

Reply:
I've also hit the "reload" icon near your name in the "reviewer" section, not sure it was needed or not to allow you to re-review